cmd/kava/cmd/root.go

```diff
@@ -85,6 +86,10 @@ func NewRootCmd() *cobra.Command {
 	return rootCmd
 }
 
+func addModuleInitFlags(startCmd *cobra.Command) {
+	crisis.AddModuleInitFlags(startCmd)
+}
+
```

Comment:
any reason to add this func when `appCreator.addStartCmdFlags` already exists?

I don't have a strong preference which to keep, but probably shouldn't have both.

Reply:
ah missed there's already a func for it, removed the new one